### PR TITLE
Set some optional parameters to be actually optional

### PR DESCRIPTION
**Pull request by Firenter**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [ ] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Run `tsc` without errors.
- [ ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

File: openlayers/index.d.ts [hunk 1]
```diff
@@ -11695,7 +11695,7 @@ declare module olx {
             resolution: number;
             start?: number;
             duration?: number;
-            easing: ((t: number) => number);
+            easing?: ((t: number) => number);
```
Comment (osechet) [inline openlayers/index.d.ts]: This change (and all the following 'easing' changes) have been done in a more recent version of ol (v3.20.0). Please update the version of the definition file accordingly.

File: openlayers/index.d.ts [hunk 2]
```diff
@@ -11979,8 +11979,8 @@ declare module olx {
          *     decimals: (number|undefined)}}
          */
         interface WriteOptions {
-            dataProjection: ol.ProjectionLike;
-            featureProjection: ol.ProjectionLike;
+            dataProjection?: ol.ProjectionLike;
```
Comment (osechet) [inline openlayers/index.d.ts]: This change does not correspond to the documentation. A better modification would be to change the `ProjectionLike` type as Projection | string | null: see [official doc](https://openlayers.org/en/latest/apidoc/ol.html#.ProjectionLike)

**Review by osechet — CHANGES_REQUESTED** [review]
Please check my comments.

**Review by osechet — APPROVED** [review]

**Comment by horiuchi** [comment]
@Firenter Thanks!

**Pull request merged**